src/stylesheets/main.less

```diff
@@ -1,4 +1,28 @@
-body {
-  color: white;
-  background: black;
+@import "colors.less";
+
+body html {
+  width : 100%;
+  height: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
```

Comment:
i'm not super crazy about adding `display` and formatting rules to `body`. Either wrap the individual components with the existing bootstrap containers, or add the rules to the components themselves.

Also, if the goal of this is to just have the divs flow down, doesn't that naturally happen with divs anyways?